exercises/sublist/src/example.c

```diff
@@ -0,0 +1,62 @@
+#include "sublist.h"
+#include <stdbool.h>
+#include <string.h>
+
+static bool is_equal(int *listOne, int *listTwo, size_t listOneLength,
+                     size_t listTwoLength)
+{
+   return (listOneLength == listTwoLength
+           && !memcmp(listOne, listTwo, listOneLength));
```

Comment:
Passing null pointers to memcmp works to an extent, but it's undefined.

Reply:
I think a more explicit check for null is in order then.